### DISCUSSION TITLE
双阿扑池子要来了

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#390] [General]
🐶日的 mhy ，顺便抽卡逻辑可能也得改了

**Comment by Arondight** [comment]
> 🐶日的 mhy ，顺便抽卡逻辑可能也得改了

希望这破游戏公司做个人， json 的结构和字段不要变

**Comment by mark9804** [comment]
除了保底次数共通之外基本上可以看作两个独立的卡池，不过可能切换卡池的时候要加一个`角色2`？

**Reply by Arondight** [reply]
但是他又不是一个常驻的规则，我想到时候看看 json ，能合并的话我想给合并了

**Comment by Arondight** [comment]
看了下规则，代码写起来稍微麻烦点，得需要两个池子，不能合并，例如池子可以叫复刻，如果 json 没啥改动那就谢天谢地了，我可以慢慢写

**Reply by mark9804** [reply]
https://webstatic.mihoyo.com/hk4e/gacha_info/cn_gf01/gacha/list.json 当中的`list`是一个array，新卡池应该也是加到这里面的（希望是），就是不知道`"gacha_name"`字段会是什么

早上看到图我还不信，说米家文案再拉也不会直接把第二个卡池叫做祈愿2啊，一看就是假消息，就是整的还挺真

**Comment by Arondight** [comment]
我看到原作者好像准备第一时间去做，我得上班做起来肯定没他快，到时候直接扒代码吧 https://github.com/SilveryStar/Adachi-BOT/issues/90

**Comment by mark9804** [comment]
可以开抄了

https://github.com/SilveryStar/Adachi-BOT/commit/58b4ea6ecb1b2f11ed50834ba7a820f11653ef12
